apps/hash-ai-worker-ts/src/workflows/run-flow-workflow.ts

```diff
@@ -387,13 +375,7 @@ export const runFlowWorkflow = async (
   if (stepWithSatisfiedDependencies.length === 0) {
     const errorMessage =
       "No steps have satisfied dependencies when initializing the flow.";
-    throw new Error(errorMessage, {
-      cause: {
-        code: StatusCode.FailedPrecondition,
-        message: errorMessage,
-        contents: [{ flow }],
-      },
-    });
+    throw new Error(errorMessage);
```

Comment:
Maybe we can discuss this internally rather than in a thread, but these outputs have been useful for me in the past when debugging errors in flows - but it makes sense to remove them if they're leading to the flow not being failed correctly.

I suppose I can use the `getFlowRuns` resolver in the meantime.

Reply:
I did this in the next PR, https://github.com/hashintel/hash/pull/4485/commits/b211df26720c762258bb7b1e25f295645d46b3e6

Reply:
Gotcha, thanks!